lib/p2p/Pool.ts

```diff
@@ -17,10 +17,18 @@ import assert from 'assert';
 import { ReputationEvent } from '../types/enums';
 
 type PoolConfig = {
+  /** Turn on/off automatically detecting and sharing current external ip address on startup. */
```

Comment:
I like the longer description here, but how about instead "turn on/off automatically..." we say "Whether to automatically..."

Reply:
consider it done